### PR TITLE
Fix bad component keys in FormInputGroup

### PR DESCRIPTION
**Pull request by davezuko**
Fixes https://github.com/OfficeDev/microsoft-teams-ui-component-library/issues/49. More work should be done in this component to implement truly _correct_ behavior, but this reasonably addresses the immediate issue.

### CODE REVIEW

File: src/components/Form/Form.tsx [hunk 1]
```diff
@@ -344,14 +344,16 @@ const TextInputsGroup = ({
...
 
-  const groupId = uniqueId("text-inputs-group");
-
   return (
     <>
       {rows.map((rowFields, r) => {
+        // TODO: row should have a stable field to use as the key, since the key
+        // will be incorrect if the rows are shuffled. I've used the index for
+        // now since it's more (but not totally) correct than the previous
+        // behavior of using a generated id that changed on each render.
```
Comment (davezuko) [inline src/components/Form/Form.tsx:353]: I looked at `rowFields` and there didn't appear to be such a key. Hence the TODO.

**Comment by akatrukhin** [comment]
LGTM

**Review by akatrukhin — APPROVED** [review]

File: src/components/Form/Form.tsx [hunk 2]
```diff
@@ -452,9 +453,10 @@ const TextInputsGroup = ({
                               .join(" ")}
                             value={formState[inputId] as string}
                             onChange={(e, props) => {
-                              if (props?.value)
+                              if (props && "value" in props) {
                                 formState[inputId] = props.value.toString();
```
Comment (davezuko) [inline src/components/Form/Form.tsx:457]: Heads up, I'm not sure we actually want to `toString()` everything, but I left the existing behavior. Can talk about this outside of this PR.

**Review by thure — APPROVED** [review]
LGTM, thanks!

**Comment by thure** [comment]
Ah shoot, we forgot to `yarn change`… hopefully the fix propagates to NPM when we push a fix anyway…

**Comment by davezuko** [comment]
@thure , want/need me to push another commit to master? I was unaware of `yarn change`.

**Pull request merged**